README.md

```diff
@@ -154,3 +154,27 @@ Examples of invalid code
 
 <div style={{ border: `1px solid ${isMagic ? "red" : "blue"}` }} />
 ```
+
+### always-spread-jsx-props-first
+
+Spreading props is a common pattern in React, but it can also lead to unintended behavior if not used carefully.
+
+By putting the spread props first, we can avoid unintended behavior, such as overriding props with the same name.
+
+This rule forces JSX spread props to always be the first prop in the list.
+
+Examples of valid code
```

Comment:
If we want to keep with the terminology and format that eslint uses we should use "correct"/"incorrect" and have the incorrect example first

Reply:
Maybe, not in this PR though